src/wolfio.c

```diff
@@ -238,6 +238,11 @@ int EmbedSend(WOLFSSL* ssl, char *buf, int sz, void *ctx)
     int sd = *(int*)ctx;
     int sent;
 
+#ifdef MAX_SEND_SZ
+    if (sz > MAX_SEND_SZ)
```

Comment:
If this is externally exposed as a macro we need add a WOLF prefix or something to prevent name clashes.